server/server.ts

```diff
@@ -52,7 +52,8 @@ if (featureSwitches.cspSecurityAudit) {
 		'report-uri /api/csp-audit-report-endpoint',
 		'report-to csp-endpoint',
 		`default-src ${cspDefaultSrcAllowList}`,
-		`default-style 'unsafe-inline'`, // this is unsafe but needed for now for emotion
+		`style-src 'unsafe-inline'`, // this is unsafe but needed for now for emotion
+		`style-src-elem 'unsafe-inline'`, // this is unsafe but needed for now for emotion
```

Comment:
you don't need this line because
"If this directive is absent, the user agent will look for the style-src directive"

https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/style-src-elem#:~:text=If%20this%20directive%20is%20absent%2C%20the%20user%20agent%20will%20look%20for%20the%20style%2Dsrc%20directive

Reply:
```suggestion
```